src/main/kotlin/nexters/linkllet/member/domain/Feedback.kt

```diff
@@ -0,0 +1,41 @@
+package nexters.linkllet.member.domain
+
+import nexters.linkllet.common.domain.BaseTimeEntity
+import javax.persistence.CascadeType
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+
+@Entity
+class Feedback(
```

Comment:
테스트 데이터 setup truncate 과정에서 `@Table` 어노테이션명을 이용하여 셋업하기 때문에 아래와 같이 수정돼야 할 것 같아요

```suggestion
@Entity
@Table(name = "feedback")
class Feedback(
```

Reply:
해당 애너테이션 추가해두겠습니닷!